packages/webextension/index.js

```diff
@@ -0,0 +1,58 @@
+const web = require('@neutrinojs/web');
+const webExt = require('web-ext').default;
+const { join } = require('path');
+
+const MODULES = join(__dirname, 'node_modules');
+
+module.exports = (neutrino, options = {}) => {
+  neutrino.use(web, options);
+
+  if (neutrino.options.debug) {
+    webExt.util.logger.consoleStream.makeVerbose();
+  }
+
+  neutrino.config
+    .output
+      .delete('chunkFilename')
+      .hotUpdateChunkFilename('hot/hot-update.js')
+      .hotUpdateMainFilename('hot/hot-update.json')
+      .end()
+    .devServer
+      .clear()
+      .end()
+    .module
+      .rules
+        .delete('worker')
+        .end()
+      .end()
+    .plugins
+      .delete(options.pluginId || 'html')
+      .end()
+    .resolve
+      .modules
+        .add(MODULES)
+        .end()
+      .end()
+    .resolveLoader
+      .modules
+        .add(MODULES)
+        .end()
+      .end()
+    .when(neutrino.options.command === 'start', () => {
+      webExt.cmd.run({
```

Comment:
This returns a promise which could be rejected in some situations. I don't see `.catch()` handled. At the very least it should log the error because I don't think web-ext will do that in the `run()` function. You could test it by omitting `sourceDir` because that should reject the promise with some kind of error.

Reply:
Agreed. I added a `.catch()` in my last commit.